### ISSUE TITLE
Grails 2.5.3 create-app fails

### ISSUE THREAD

**Issue by pawelhalasa** [grails/grails-core#9450]
When creating a new app (`grails create-app test`), Grails 2.5.3 fails with the following error message:
```
| Error Resolve error obtaining dependencies: Failed to resolve dependencies (Set log level to 'warn' in BuildConfig.groovy for more informa
tion):
- org.aspectj:aspectjweaver:1.8.5
```

**Comment by graemerocher** [comment]
hmm looks like you'll need to create the app with 2.5.2 then update the Grails version, thanks for the report

**Comment by pjungermann** [comment]
could that be related to the change of the spring version from 4.1.6.RELEASE to 4.1.8.RELEASE? At "aspectjVersion" in build.gradle is a comment that the version has to be the same as in org.springframework:spring-aspects and that is 1.8.7 and not 1.8.5.

There have not been many changes from 2.5.2 to 2.5.3..

**Comment by graemerocher** [comment]
Yes it is due to the Spring upgrade

**Comment by pjungermann** [comment]
@graemerocher : wouldn't do changing aspectjVersion to 1.8.7 do the trick then? I could prepare the PR for it

**Comment by graemerocher** [comment]
yes that would do the trick :)

**Comment by pjungermann** [comment]
PR #9563 for 2.5.x

**Comment by graemerocher** [comment]
Thanks!

**Comment by pjungermann** [comment]
@graemerocher I also created PR #9564 for 3.0.x in case you want to have the version change for it as well

**Comment by graemerocher** [comment]
Thanks, the issue doesn't impact 3.0.x but it will be useful nonetheless

**Comment by nickdos** [comment]
I've just hit this bug. I downloaded the most recent 2.5.x version available on the Grails site (2.5.3) and ran `grails create-app foo` and it gives the errors listed above (using Java 1.8.0_31).

I see this is fixed in milestone 2.5.4 but that version is not listed on the Grails download page. Is there a reason for this?
